### PR TITLE
Object assign issue

### PR DESCRIPTION
**Pull request by gregory-latinier**

### CODE REVIEW

**Comment by gregory-latinier** [comment]
I'll investigate and if it's object assign related I'll work on this PR otherwise I'll create another one for the safari

File: views/index.hbs [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <meta id="viewport" name="viewport" content="initial-scale=1.0, maximum-scale=1.0, user-scalable=0"/>
   <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
   <link rel="stylesheet" href="/css/base.css" type="text/css" media="all"/>
+  <script src="//cdn.polyfill.io/v2/polyfill.min.js?features=Intl.~locale.en,Intl.~locale.fr,Intl.~locale.ko,Intl.~locale.ru,Intl.~locale.zh"></script>
```
Comment (bonustrack) [inline views/index.hbs]: We should no use external dependencies like this, can we have it localy?

Comment (gregory-latinier) [reply views/index.hbs]: The advantage of this method is that this service detect is the browser needs to load something.
The JS is empty if the browser supports Intl.
Should I still find a way to provide it locally ?

Comment (bonustrack) [reply views/index.hbs]: You should find another solution. `polyfill.io` is not a website i want to trust.

Comment (bonustrack) [reply views/index.hbs]: We should not rely on external code

Comment (gregory-latinier) [reply views/index.hbs]: Sure I'll work on that

**Pull request merged**